static/src/javascripts/projects/common/modules/article/spacefinder.js

```diff
@@ -175,7 +175,7 @@ define([
         var body;
 
         rules = rules || defaultRules;
-        body = rules.bodySelector ? document.querySelector(rules.bodySelector) : document;
+        body = (rules.bodySelector && document.querySelector(rules.bodySelector)) || document;
```

Comment:
Now I think about this, I am probably hiding another problem - `document.querySelector(rules.bodySelector)` was returning null, I guess because this JS was being run maybe when it shouldn't (live blog template). Efficiency wise, [the tenerary looks faster](https://jsperf.com/ternary-vs-double-ampersand-and-or), so that's even more reason to find the root cause.